src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -49,8 +51,8 @@ public PersistentDataStore() {
   /**
    * Loads all User objects from the Datastore service and returns them in a List.
    *
-   * @throws PersistentDataStoreException if an error was detected during the load from the
-   *     Datastore service
+   * @throws codeu.model.store.persistence.PersistentDataStoreException if an error was detected
```

Comment:
I think this comment should stay the same (here and other places)

Reply:
I'm not sure if the auto-format added the "codeu.model.store.persistence" part to the comment. But it sees to only be an issue in PersistentDatatstore